source/constructs/api/data_source/service.py

```diff
@@ -361,13 +381,338 @@
+                            logger.info(up_cr_response)
+                    except Exception as e:
+                        logger.info("update_crawler error")
+                        logger.info(str(e))
+                    st_cr_response = glue.start_crawler(
+                        Name=crawler_name
+                    )
+                    logger.info(st_cr_response)
+                except Exception as e:
+                    logger.info("sync_rds_connection get_crawler and create:")
+                    logger.info(str(e))
+                    response = glue.create_crawler(
+                        Name=crawler_name,
+                        Role=crawler_role_arn,
+                        DatabaseName=glue_database_name,
+                        Targets={
+                            'JdbcTargets': jdbc_targets,
+                        },
+                        Tags={
+                            'AdminAccountId': _admin_account_id
+                        }
+                    )
+                    logger.info(response)
+                    start_response = glue.start_crawler(
+                        Name=crawler_name
+                    )
+                    logger.info(start_response)
+                crud.create_jdbc_connection(jdbc.account_provider,
+                                            jdbc.account_id,
+                                            jdbc.region,
+                                            jdbc.instance,
+                                            glue_connection_name,
+                                            glue_database_name,
+                                            None,
+                                            crawler_name)
+            else:
+                crud.set_jdbc_connection_glue_state(jdbc.account_provider, jdbc.account_id, jdbc.region, jdbc.instance,
+                                                    MessageEnum.SOURCE_JDBC_NO_SCHEMA.get_msg())
+                raise BizException(MessageEnum.SOURCE_JDBC_NO_SCHEMA.get_code(), MessageEnum.SOURCE_JDBC_NO_SCHEMA.get_msg())
+    except Exception as err:
+        crud.set_jdbc_connection_glue_state(jdbc.account_provider, jdbc.account_id, jdbc.region, jdbc.instance, str(err))
+        glue = boto3.client('glue',
+                            aws_access_key_id=credentials['AccessKeyId'],
+                            aws_secret_access_key=credentials['SecretAccessKey'],
+                            aws_session_token=credentials['SessionToken'],
+                            region_name=jdbc.region
+                            )
+        """ :type : pyboto3.glue """
+        try:
+            glue.delete_crawler(crawler_name)
+        except Exception:
+            pass
+        try:
+            glue.delete_database(Name=glue_database_name)
+        except Exception:
+            pass
+        try:
+            glue.delete_connection(ConnectionName=glue_connection_name)
+        except Exception:
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

Try, Except, Pass detected. https://bandit.readthedocs.io/en/latest/plugins/b110_try_except_pass.html